README.md

```diff
@@ -1,33 +1,74 @@
-# `Turborepo` Vite starter
+# CLASS-web
 
-This is an official starter Turborepo.
+This is an implementation of CLASS that runs entirely in the browser.
 
-## Using this example
+The CLASS web application is available at https://classmodel.github.io/class-web.
 
-Run the following command:
+For more information on CLASS, see https://classmodel.github.io/.
 
-```sh
-npx create-turbo@latest -e with-vite
-```
+## Developers
+
+This repository is a so-called monorepo, where multiple packages and application
+can easily be developed in tandem.
+
+We used [TurboRepo](https://turbo.build/repo) for the initial setup, which uses
+[pnpm workspaces](https://pnpm.io/workspaces) under the hood. As such, it is
+possible to do advanced tricks with Turbo, such as "lint/test/build all
+apps/packages at once with `turbo build`", and share tooling configurations
+across packages/apps, but since this repo is small, we will not rely too much on
+these features.
+
+Currently the repo is home to the following:
 
-## What's inside?
+- packages/
+  - class: reimplementation of CLASS in typescript
+  - config-eslint: shared configuration for eslint
+  - config-typescript: shared configuration for typescript
+- apps/
+  - class-solid: web application with a graphical user interface for CLASS
 
-This Turborepo includes the following packages and apps:
+## Local build
+
+To run a local development version:
+
+```sh
+git clone git@github.com:classmodel/class-web.git
+cd class-web/apps/class-solid
+pnpm install
+pnpm run dev
```

Comment:
Can be simplified to

```suggestion
cd class-web
pnpm install
pnpm dev
```

This also makes sure that if a package needs building, that that is done before the app is ready. Thanks to turborepo